xarray/core/variable.py

```diff
@@ -1334,7 +1334,7 @@ def where(self, cond, other=dtypes.NA):
         return ops.where_method(self, cond, other)
 
     def reduce(self, func, dim=None, axis=None,
-               keep_attrs=None, allow_lazy=False, **kwargs):
```

Comment:
Is there a reason this isn't default `False`?
(I'm not sure why `keep_attrs` is default `None`, but looks like it's calling into `_get_keep_attrs` at least)

Reply:
Yes it was to follow the behaviour of `keep_attrs`, happy to change the default to be `False`